vendor/cppbackport-master/lib/fs/rename.cpp

```diff
@@ -45,7 +45,17 @@ void rename(const path& from, const path& to)
 		if (copy_file(from, to))
 			cpp17::filesystem::remove(from);
 	}
-}
+} // end rename
+
+void rename(const path& from, const path& to, __attribute__((unused)) std::error_code& ec)
```

Comment:
what is the point of this function providing a `std::error_code` if it doesn't even set it on error?